pyproject.toml

```diff
@@ -38,7 +38,6 @@ addopts = "--jaxtyping-packages=quax,beartype.beartype(conf=beartype.BeartypeCon
 [tool.ruff.lint]
 select = ["E", "F", "I001"]
 ignore = ["E402", "E721", "E731", "E741", "F722"]
-ignore-init-module-imports = true
```

Comment:
Why remove this?

Reply:
Newer `ruff` doesn't require it. It surfaces a warning saying that it's being deprecated.